bower-publish.sh

```diff
@@ -15,13 +19,13 @@ rm -rf bower-chosen/*
 cp public/bower.json public/*.png public/chosen.jquery.js public/chosen.css bower-chosen/
 cd bower-chosen
 
-LATEST_VERSION=$(git diff bower.json | grep version | cut -d':' -f2 | cut -d'"' -f2 | tail -1)
+LATEST_VERSION=`git tag -l "*.*.*" | sed 's/^v//' | sort -V | tail -1`
 
-if [ -z $LATEST_VERSION ] ; then
+if [ "$LATEST_VERSION" = "$CHOSEN_VERSION" ] ; then
   echo "No Chosen version change. Skipped tagging"
 else
-  echo "Chosen version changed. Tagging version ${LATEST_VERSION}\n"
-  git tag -a "v${LATEST_VERSION}" -m "Version ${LATEST_VERSION}"
+  echo "Chosen version changed. Tagging version ${CHOSEN_VERSION}\n"
+  git tag -a "${CHOSEN_VERSION}" -m "Version ${CHOSEN_VERSION}"
```

Comment:
#2446 is actually a bugfix, the previous behavior was unintended. 

But those mobile browsers.. Yeah probably should be `v2.0.0`. But I think we should a `v1.4.3` along that, to fix the bower issue for `v1.4.2`